deploy_local.sh

```diff
@@ -6,3 +6,4 @@ source venv/bin/activate
 
 # Install dependencies
 pip install --no-cache-dir -r requirements.txt
+pip install --no-cache-dir -r dev-requirements.txt
```

Comment:
I see that we are using requirements.txt which is fairly standard with `pip`. I believe that there may be better alternatives however.. In the past, quite a few years ago, I have seen `pipenv` be a bit more robust, but I am seeing here that there are other options, seemingly (https://www.reddit.com/r/Python/comments/16qz8mx/pipenv_piptools_pdm_or_poetry/) `poetry` having gained some traction over the years:
<img width="2047" alt="Screenshot 2024-08-12 at 12 04 18 PM" src="https://github.com/user-attachments/assets/0c99d6cb-9b05-42ac-8584-cd97fb7e3764">
it may be worth looking for alternatives if the team finds ourselves in the position where we are consistently dealing with an evolving code base written in Python.

Reply:
I will have to look into it a bit more. But I might be worth creating a separate task for that. What do you think?

Reply:
oh sorry, I should have clarified, this comment was meant to be more of something to keep in mind in case we are continuously working with Python... if the payment module hardly sees any code changes and is working for us as it is, there is no need to make any changes 👍